.env

```diff
@@ -0,0 +1,4 @@
+REACT_APP_ORDERS_CONTRACT_ADDRESS=0xE58E94E87547A4FfE03f11Ee086adc31cEED3F03
+REACT_APP_BLOCK_EXPLORER_TRANSACTION_URL=https://goerli.etherscan.io/tx
+REACT_APP_PROVIDER_ALCHEMY_PUBLIC_API_KEY=dZfJX0zlMxU1bwn_91-6KYdVb4m5V2hs
+REACT_APP_PROVIDER_ALCHEMY_NETWORK_NAME=goerli
```

Comment:
This was a "bonus" refactor, to move hard-coded test/staging strings into .env (which is supported out-of-the-box by create-react-app scaffolding tool)